docs/contracts.rst

```diff
@@ -473,7 +473,7 @@ The following statements are considered modifying the state:
     }
 
 .. note::
-  ``constant`` on functions is an alias to ``view``, but this is deprecated and will be dropped in version 0.5.0.
+  ``constant`` on functions used to be an alias to ``view``, but this was dropped in version 0.5.0.
```

Comment:
I think we need to be a bit more explicit here. Such as "it cannot be used on functions anymore".